lib/rmt/scc.rb

```diff
@@ -51,7 +51,9 @@ def export(path)
     scc_api_client = SUSE::Connect::Api.new(Settings.scc.username, Settings.scc.password)
 
     @logger.info('Exporting products')
-    File.write(File.join(path, 'organizations_products.json'), scc_api_client.list_products.to_json)
+    File.write(File.join(path, 'organizations_products_scoped.json'), scc_api_client.list_products.to_json)
+    # For SUMA, we also export the unscoped products with the filename it expects.
+    File.write(File.join(path, 'organizations_products.json'), scc_api_client.list_products_unscoped.to_json)
```

Comment:
Are those the names SUMA expects? Just thinking that it is quite confusing to see _scoped there. If I am a customer, I can ask - how they were scoped?

Reply:
@kalabiyau SUMA expects unscoped products in `organization_products.json`, please see [the thread above](https://github.com/SUSE/rmt/pull/61#discussion_r158025697).